src/screens/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,198 @@
+import { useState } from "react";
+import { View, Text, Image } from "react-native";
+import uuid from "react-native-uuid";
+import { useForm } from "@hooks/useForm";
+import AsyncStorageManager from "@storage/AsyncStorageManager";
+import TextField from "@components/TextField";
```

Comment:
where is `components` folder??

Reply:
My custom components branch is merged to dev and this branch is also created from dev
so all the components are present but are not in this PR